src/project-scanner.ts

```diff
@@ -73,21 +75,47 @@ export class ProjectScanner {
     return packageScanner.getConversionManifest();
   }
 
+  /**
+   * Reads the entrypoints for a package from the `main` field in the package's
+   * 'bower.json' file.
+   */
+  private async readBowerJsonEntrypoints(packageName: string) {
+    const bowerJsonPath =
+        this.urlHandler.packageRelativeToOriginalUrl(packageName, 'bower.json');
+    const bowerJson = await fse.readJSON(bowerJsonPath) as Partial<BowerConfig>;
```

Comment:
`fse.readJSON` rejects when the file doesn't exist, so this needs to be wrapped in a try/catch which sets `bowerMainFiles` to `[]` with a console warning?

Reply:
fixed